.github/workflows/test.yml

```diff
@@ -78,6 +78,11 @@ jobs:
         with:
           cluster_name: kind
       
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.10.0
```

Comment:
Let's remove this parameter and just use the latest. This way we don't need to maintain this version number.

Reply:
(scenario: [nonroot, rootuser, allroot]) -Even I too felt the same thought of checking and removing "allroot" scenario.
All root is not needed we hard coded security context in library so we need non-root and rootuser.(If we need any consumer application with root user otherwise we leave with nonroot user)

passing latest version and updating PR now.

Reply:
-Updated latest version
-Now removed allroot scenario and currently they are nonroot and rootuser.